perma_web/api/tests/test_link_authorization.py

```diff
@@ -122,6 +122,15 @@ def test_should_allow_member_of_links_org_registrar_to_patch_notes_and_title(sel
     def test_should_reject_patch_from_user_lacking_owner_and_folder_access(self):
         self.rejected_patch(self.link_url, user=self.unrelated_org_member, data=self.patch_data)
 
+    def test_should_allow_user_to_patch_with_file(self):
```

Comment:
Can we add a `test_should_reject_patch_with_file_for_out_of_window_link`, like the current `test_should_reject_delete_for_out_of_window_link`?